explorer/src/cli/clnodes.ts

```diff
@@ -0,0 +1,78 @@
+import { default as fetch, Response } from 'node-fetch'
+import httpStatus from 'http-status-codes'
+import {
+  ADMIN_USERNAME_HEADER,
+  ADMIN_PASSWORD_HEADER,
+} from '../utils/constants'
+
+const EXPLORER_BASE_URL =
+  process.env.EXPLORER_BASE_URL || 'https://explorer.chain.link'
+const EXPLORER_ADMIN_USERNAME = process.env.EXPLORER_ADMIN_USERNAME
+const EXPLORER_ADMIN_PASSWORD = process.env.EXPLORER_ADMIN_PASSWORD
+
+interface CreateChainlinkNode {
+  name: string
+  url?: string
+}
+
+interface CreateChainlinkNodeOk {
+  id: string
+  accessKey: string
+  secret: string
+}
+
+export const add = async (name: string, url?: string) => {
+  const createNodeUrl = `${EXPLORER_BASE_URL}/api/v1/admin/nodes`
+  const data: CreateChainlinkNode = { name: name }
+  if (url) {
+    data.url = url
```

Comment:
no need for the conditional here, we can include it in the definition of `data` since its optional anyway